Sources/PackageDescription/PackageDescription.swift

```diff
@@ -185,19 +184,19 @@ public final class Package {
     }
     private var _defaultLocalization: LanguageTag?
 
-    /// The name to use for C Modules.
+    /// The name to use for C modules.
     ///
-    /// If present, the Swift Package Manager searches for a `<name>.pc` file
+    /// If present, the Swift package manager searches for a `<name>.pc` file
     /// to get the required additional flags for a system target.
     public var pkgConfig: String?
 
-    /// An array of providers for the system target.
+    /// An array of providers for a system target.
     public var providers: [SystemPackageProvider]?
 
-    /// The list of targets.
+    /// The list of targets that are part of this package.
     public var targets: [Target]
 
-    /// The list of products that this package vends and that can be run or used by its clients.
```

Comment:
I don't think this is correct, clients can't actually run executable products today.